modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2160,6 +2251,62 @@ scheduleTest.describe('Manage object entries schedule properties', () => {
 		}
 	);
 
+	scheduleTest(
+		'can see approved and scheduled labels for entry with a display date versioning enabled and at least one version approved',
+		async ({apiHelpers, page, viewObjectEntriesPage}) => {
+			const objectDefinitionAPIClient =
+				await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+			await objectDefinitionAPIClient.patchObjectDefinition(
+				_objectDefinition.id,
+				{
+					enableObjectEntryVersioning: true,
+				}
+			);
+
+			await viewObjectEntriesPage.goto(_objectDefinition.className);
+
+			await viewObjectEntriesPage.clickAddObjectEntry(
+				_objectDefinition.label['en_US']
+			);
+
+			await viewObjectEntriesPage.choosePublicationOption('publish');
+
+			await waitForAlert(page);
+
+			await viewObjectEntriesPage.backButton.click();
+
+			await expect(page.getByText('Approved')).toBeVisible();
+
+			await expect(page.getByText('Scheduled')).not.toBeVisible();
+
+			await page
+				.locator('.cell-id')
```

Comment:
could we try use the cell role?

Reply:
hey Igor, I invested a little more time and found a way to get the entryId, please take another look when you can!